server/server_test.go

```diff
@@ -11675,19 +11684,28 @@ stream
 				}},
 				TICKscript: tick,
 				Status:     client.Enabled,
-			}); err != nil {
+			})
+			if err != nil {
 				t.Fatalf("%s: %v", kind, err)
 			}
 
+			if err != nil {
+				t.Fatalf(" Error deleting task: %s: %v", kind, err)
```

Comment:
This looks like unreachable code.  The check for `err != nil` above means this can never execute.  Also, the error is in task creation, not task deletion.